### PR TITLE
Add link to video showing imaging step-by-step

### PR DESCRIPTION
**Pull request by horner**
This video is really good at showing the steps to imagine the Rio 2

### CODE REVIEW

File: source/docs/hardware/hardware-basics/status-lights-ref.rst [hunk 1]
```diff
@@ -27,6 +27,8 @@ roboRIO
   :alt: roboRIO status lights highlighted.
   :width: 400
 
+`Imaging the RoboRIO 2 Video <http://www.youtube.com/watch?v=v=jvd-Pxaqy9I>`_
```
Comment (sciencewhiz) [inline source/docs/hardware/hardware-basics/status-lights-ref.rst:30]: Why put the video here?

File: source/docs/zero-to-robot/step-3/imaging-your-roborio.rst [hunk 2]
```diff
@@ -7,6 +7,9 @@ Imaging your roboRIO 1
 
 .. note:: The roboRIO 2 uses different imaging instructions. The imaging instructions for the NI roboRIO 2.0 are :doc:`here<roborio2-imaging>`.
 
+`Imaging the RoboRIO 2 Video <http://www.youtube.com/watch?v=v=jvd-Pxaqy9I>`_
```
Comment (sciencewhiz) [inline source/docs/zero-to-robot/step-3/imaging-your-roborio.rst:10]: The video should be on the roboRIO 2 imaging page, not this one

**Review by sciencewhiz — CHANGES_REQUESTED** [review]
The video shows as not available for me

**Comment by horner** [comment]
That's odd, I'll update it. Does this work for you:

https://youtu.be/jvd-Pxaqy9I

**Comment by sciencewhiz** [comment]
Yes, that link works for me.

See this for how we embed videos: https://github.com/wpilibsuite/frc-docs/blob/01d4af7b098486bdf4ca2b388a575f6cf7c88906/source/docs/software/advanced-controls/system-identification/running-routine.rst?plain=1#L22

**Pull request merged**